modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -142,10 +152,10 @@
 	alt_titles = list("Shaft Miner", "Excavator", "Spelunker", "Drill Technician", "Prospector")
 
 /datum/job/station_engineer
-	alt_titles = list("Station Engineer", "Emergency Damage Control Technician", "Electrician", "Engine Technician", "EVA Technician", "Engineer", "Mechanic")
+	alt_titles = list("Station Engineer", "Emergency Damage Control Technician", "Electrician", "Engine Technician", "EVA Technician", "Mechanic")
 
 /datum/job/virologist
 	alt_titles = list("Virologist", "Pathologist")
 
 /datum/job/warden
-	alt_titles = list("Warden", "Brig Sergeant", "Dispatch Officer", "Brig Governor", "Jailer", "Gaoler")
+	alt_titles = list("Warden", "Brig Sergeant", "Dispatch Officer", "Brig Governor", "Jailer", "Brig Gaoler")
```

Comment:
I've talked with Tom and he's not overly attached, and I really think this makes it a lot less clear who they are, so I don't think it's a good addition.
```suggestion
	alt_titles = list("Warden", "Brig Sergeant", "Dispatch Officer", "Brig Governor", "Jailer")
```